tests/unit/taqueria-sdk/data/v2/.taq/config.local.testing.json

```diff
@@ -1,6 +1,6 @@
 {
 	"networkName": "ghostnet",
-	"aliases": {
+	"contracts": {
```

Comment:
Can you try originate and then transfer? reason being that originate will create the alias (`counter` for `counter.tz`) and when you call transfer using `counter`, it will look for the `alias` field in the config. Just making sure this works.

Reply:
Appears to work fine @jchenche:

```
MacBook-Pro:test-project mweichert$ taq originate counter.tz
┌────────────┬──────────────────────────────────────┬─────────┬──────────────────┬────────────────────────┐
│ Contract   │ Address                              │ Alias   │ Balance In Mutez │ Destination            │
├────────────┼──────────────────────────────────────┼─────────┼──────────────────┼────────────────────────┤
│ counter.tz │ KT1MyCnYS9JfHAGCE8TWkMrdSEzQDzDSRjVZ │ counter │ 0                │ http://localhost:20000 │

MacBook-Pro:test-project mweichert$ taq transfer counter --param counter.parameter.increment_by_3.tz
┌────────────────┬──────────────────────────────────────┬──────────────────┬────────────┬────────────────┬────────────────────────┐
│ Contract Alias │ Contract Address                     │ Parameter        │ Entrypoint │ Mutez Transfer │ Destination            │
├────────────────┼──────────────────────────────────────┼──────────────────┼────────────┼────────────────┼────────────────────────┤
│ counter        │ KT1MyCnYS9JfHAGCE8TWkMrdSEzQDzDSRjVZ │ (Left (Right 3)) │ default    │ 0              │ http://localhost:20000 │
│                │                                      │                  │            │                │                        │
└────────────────┴──────────────────────────────────────┴──────────────────┴────────────┴────────────────┴────────────────────────┘
```